lib/krane/cluster_resource_discovery.rb

```diff
@@ -52,9 +52,20 @@ def fetch_mutating_webhook_configurations
 
     private
 
+    # During discovery, the api paths may not actually be at the root, so we must programatically find it.
+    def base_api_path
+      @base_api_path ||= begin
+        raw_response, err, st = kubectl.run("config", "view", "--minify", "--output",
```

Comment:
so... how does this work when `kubeconfig` doesn't have a `current-context` set? or we are setting the currently config explicitly somewhere else before this runs?

Reply:
looks like you are still able to provide `--context` when using the `config` command. ie, 

```
# using different random names
kubectl --context my-context-38 config view --minify --output jsonpath="{.clusters[*].cluster.server}"
https://35.x.x.x%

kubectl --context my-context-37 config view --minify --output jsonpath="{.clusters[*].cluster.server}"
https://34.x.x.x%
```

you get the gist, missing explicit argument basically.

Reply:
guessing I first need to read the code before 😮. Looks like the `Kubectl` class already does this by default in https://github.com/Shopify/krane/blob/master/lib/krane/kubectl.rb#L116 unless explicitly disabled.